src/jpegCompression.cpp

```diff
@@ -39,7 +39,7 @@ std::unique_ptr<uint8_t[]> JpegCompression::compress(
       channel_t, typename boost::gil::color_space_type<
                      boost::gil::rgb8_view_t>::type>::_color_space;
 
-  uint64_t outlen = 0;
+  unsigned long outlen = 0; //NOLINT, macOS workaround
```

Comment:
Why is lint complaining here? Can you use size_t like you did elsewhere?

Reply:
yep, reverted, it was workaround for different version of compiler, works fine with gcc9